src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -251,10 +257,20 @@ def create_feedback_issue(event, project_id: int, source: FeedbackCreationSource
     )
     now = datetime.now()
 
+    # Force `tags` to be a dict if it's initially a list,
```

Comment:
can we move this to inside `fix_for_issue_platform` before L149, so it's colocated w/ the code that requires a tags dict?

Also the callstack is shim -> create_feedback -> fix_for_.. so if we do it at the end, all 3 of these functions can accept either tag format.